docs/ConfiguringIssuerKit.md

```diff
@@ -0,0 +1,129 @@
+# Configuring Issuer Kit
+
+Issuer Kit can be configured and customized by modifying a few files in each of the services. The following is a list of configuration files, where they should go and what are the supported settings.
+
+## Web Applications
+
+All web applications in Issuer Kit (`issuer-admin` and `issuer-web`) are built on the same VueJS app skeleton, and therefore most of the configuration files between the two apps are in common.
+
+All files inside the `public` folder can be overridden at deployment time: in the production-ready containers for Issuer Kit the location to place the new files (overriding the existing ones with the same names) is `/srv/config`.
+
+### Common Settings
+
+##### UI Theme
+
+The UI theme for the web application can be lightly customized by tweaking the configuration in `config/vuetify.json`. The contents of this file are Vuetify heme configurations and will allow you to change the base color scheme for the app, as well as some other settings. Please refer to the [Vuetify docs](https://vuetifyjs.com/en/customization/theme/#theme) for further details. 
+
+Additionally, `favicon.ico` and `logo.svg` can be replaced to customize the browser favorite icon for the webapp and the SVG logo appearing in the header of the webapp.
+
+#### OIDC Plugin
+
+The file `silent-renew-oidc.html` is responsible for silently renewing the access token automatically when using an Identity Provider to access the webapp. Modifying it is not recommended, however further docs can be found in the documentation for [vuex-oidc](https://github.com/perarnborg/vuex-oidc/wiki).
+
+#### Unauthorized page
+
+When an unauthorized user (IdP authentication failure, no authentication or no valid invitation) tries to access the web application, the unauthorized page will be displayed. The default page is generic and only displays a basic message, to further customize the contents of this page `unauthorized.html` can be used to inject custom HTML and CSS into the page.
+
+The contents of the file will be displayed below the generic unauthorized message.
+
+#### Home page
+
+Similarly to the `unauthorized.html` configuration file we discussed earlier, `terms-and-confitions.html` defines custom HTML/CSS that will be injected in the main landing page for the issuer webapp. This can be used to add a welcome message or a disclaimer.
+
+#### Dynamic Form Configuration
+
+Both the admin and issuer webapp use SurveyJS to display forms that are dynamically generated from a configuration file.
```

Comment:
Please add link to `[SurveyJS](https://surveyjs.io)`.  Nice to note the connection between the config file referenced here and the one below.  Maybe change this to "a SurveyJS configuration file, referenced below."